### PR TITLE
Provide support for Container Credentials for SQS Channel #43

### PR DESCRIPTION
**Pull request by shekhar-dev-tibco**
Provide support for Container Credentials for SQS Channel #43

### CODE REVIEW

File: channel/aws-sqs/src/main/java/com/tibco/be/custom/channel/aws/sqs/SqsListener.java [hunk 1]
```diff
@@ -71,6 +72,9 @@ public void run() {
         } else if (context instanceof DefaultContext) {
             logger.log(Level.DEBUG,"Using Default Context");
             queueUrl = ((DefaultContext)context).getQueueUrl();
+        } else if (context instanceof ContainerContext) {
+            logger.log(Level.DEBUG,"Using Contaimer Context");
```
Comment (vpatil-tibco) [inline channel/aws-sqs/src/main/java/com/tibco/be/custom/channel/aws/sqs/SqsListener.java:76]: Typo

File: channel/aws-sqs/src/main/java/com/tibco/be/custom/channel/aws/sqs/containercredentials/ContainerContext.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
+package com.tibco.be.custom.channel.aws.sqs.containercredentials;
```
Comment (vpatil-tibco) [inline channel/aws-sqs/src/main/java/com/tibco/be/custom/channel/aws/sqs/containercredentials/ContainerContext.java:1]: Include header with necessary documentation.

**Review by vpatil-tibco — CHANGES_REQUESTED** [review]
Most of it looks ok. Some minor comments. I don't see any add/update to unit test for this. Can you include that too.

**Comment by shekhar-dev-tibco** [comment]
Incorporated review comments. Testing for the same is out of scope for dev, as it involves ECS setup with required variables for the same.

**Comment by vpatil-tibco** [comment]
For tests we use LocalStackContainer which mocks all AWS resources, so you should be able to use that to mock this option as well.

**Comment by shekhar-dev-tibco** [comment]
https://github.com/localstack/localstack
ECS is supported in pro version

**Pull request merged**